applications/seo.md

```diff
@@ -4,13 +4,13 @@ description: Machine translation for SEO
 
 ### SEO
 
-Machine translation for for **search engine optimization** (SEO) is the translation of website content into the languages in which users are searching.
+Machine translation for **search engine optimization \(SEO\)** is the translation of [commerce and marketplace](/applications/commerce-and-marketplaces) website content into the languages in which users are searching.
```

Comment:
I think the parens should not be bold:
`**search engine optimization** \(**SEO\)**`